app/uk/gov/hmrc/vatsubscription/controllers/actions/VatAuthorised.scala

```diff
@@ -41,8 +45,8 @@ class VatAuthorised @Inject()(val authConnector: AuthConnector, implicit val app
 
   def async(vrn: String)(f: User[_] => Future[Result])(implicit ec: ExecutionContext): Action[AnyContent] = Action.async {
     implicit request =>
-      authorised(delegatedAuthRule(vrn)).retrieve(Retrievals.allEnrolments and Retrievals.credentials) {
-        case enrolments ~ credentials =>
+      authorised(delegatedAuthRule(vrn)).retrieve(retrieve.allEnrolments and retrieve.credentials) {
+        case  enrolments ~ Some(credentials) =>
```

Comment:
Could you add that other scenario in case the credId isn't there please? I think it should return Forbidden, but with a WARN log message as it would be interesting to see if we get any users without it in production